### PR TITLE
refactor: add csrf faq url to error msg in local env

### PR DESCRIPTION
**Pull request by shaoshuai0102**
eggjs/egg#387

### CODE REVIEW

File: app/extend/context.js [hunk 1]
```diff
@@ -111,7 +112,12 @@ module.exports = {
     //  synchronize form requests' token always changing to protect against BREACH attacks
     if (token !== this[CSRF_SECRET] && !tokens.verify(this[CSRF_SECRET], token)) {
       debug('verify secret and token error');
-      this.throw(403, 'invalid csrf token');
+      this.throw(403, `Invalid csrf token. ${this[CSRF_NOTICE_SUFFIX]}`);
     }
   },
+
+  get [CSRF_NOTICE_SUFFIX]() {
+    if (this.app.config.env === 'local') return 'See https://eggjs.org/zh-cn/core/security.html#安全威胁csrf的防范';
```
Comment (shaoshuai0102) [inline app/extend/context.js]: what's the english doc url? @popomore

**Comment by jtyjty99999** [comment]
+1

File: app/extend/context.js (hunk 1 above)
Comment (dead-horse) [reply app/extend/context.js]: 感觉不要在 message 里面加吧，宁愿打日志，但是现在又没有英文文档。

Comment (shaoshuai0102) [reply app/extend/context.js]: just in local env, it's ok I think. This also results in logs.

We can pending this pr util this part of english doc is translated. ok?

Comment (dead-horse) [reply app/extend/context.js]: 就算在开发环境，放错误信息里面也显得不太专业，这个本来就是给服务端的人看的，客户端拿到这个信息不应该

Comment (atian25) [reply app/extend/context.js]: agree with @dead-horse, just logger.warn

Comment (shaoshuai0102) [reply app/extend/context.js]: fine

**Review by atian25 — APPROVED** [review]

**Comment by shaoshuai0102** [comment]
done

**Comment by popomore** [comment]
no test？

**Comment by shaoshuai0102** [comment]
TESTS added

**Review by popomore — APPROVED** [review]

**Comment by popomore** [comment]
@shaoshuai0102 来发个版本？

**Comment by popomore** [comment]
+ egg-security@1.6.0

**Pull request merged**